lib/faker/default/construction.rb

```diff
@@ -2,26 +2,80 @@
 
 module Faker
   class Construction < Base
+    ##
+    # Produces a random material.
+    #
+    # @return [String]
+    #
+    # @example
+    #   Faker::Construction.material #=> "Wood"
+    #
+    # @faker.version 1.9.2
     def self.material
       fetch('construction.materials')
     end
 
+    ##
+    # Produces a random heavy_equipment.
+    #
+    # @return [String]
+    #
+    # @example
+    #   Faker::Construction.heavy_equipment #=> "Excavator"
+    #
+    # @faker.version 1.9.2
     def self.heavy_equipment
       fetch('construction.heavy_equipment')
     end
 
+    ##
+    # Produces a random trade.
+    #
+    # @return [String]
+    #
+    # @example
+    #   Faker::Construction.trade #=> "Carpenter"
+    #
+    # @faker.version 1.9.2
     def self.trade
       fetch('construction.trades')
     end
 
+    ##
+    # Produces a random subcontract category.
+    #
+    # @return [String]
+    #
+    # @example
+    #   Faker::Construction.subcontract_category #=> "Curb & Gutter"
+    #
+    # @faker.version 1.9.2
     def self.subcontract_category
       fetch('construction.subcontract_categories')
     end
 
+    ##
+    # Produces a random standard_cost_code.
```

Comment:
Can you update this one?

```suggestion
    # Produces a random standard cost code.
```

Reply:
Good catch, thanks!